input/pagecontent/reference.md

```diff
@@ -2195,6 +2314,36 @@ To define a profile, the keywords `Profile` and `Parent` are required, and `Id`,
   * valueCodeableConcept from https://loinc.org/vs/LL3991-8 (extensible)
   ```
 
+#### Defining Resources
+
+Custom resources allow authors to define new structures representing arbitrary content. Resources are defined similar to [logical models](#defining-logical-models), but are intended to support data exchange using FHIR's RESTful API mechanisms. The capability to define resources may be used by HL7 to define core FHIR resources or by other organizations to define proprietary resources for their own internal use. Custom (non-HL7) resources should not be used for formal exchange between organizations; only standard FHIR resources and profiles should be used for inter-organizational exchange of health data. As such, the the FHIR IG publisher does not support including custom resources in implementation guides.
+
+Resources are defined in FSH using the keyword `Resource`. The keywords `Parent`, `Id`, `Title`, and `Description` are OPTIONAL.
+
+Only [DomainResource](http://hl7.org/fhir/R4/domainresource.html)  and [Resource](http://hl7.org/fhir/R4/resource.html) are allowed as parents of a resource. If no `Parent` is specified, DomainResource is used as the default parent.
+
+Rules defining the resource follow immediately after the keyword section. Resources are primarily comprised of AddElement rules. Since resources are represented using StructureDefinition, many of the rules used in profile definitions may also be used when defining resources. According to the FHIR specification's [interpretation of ElementDefinition in different contexts](http://hl7.org/fhir/R4/elementdefinition.html#interpretation), however, resources may not use slicing or fixed/patterned values.  As a result, Contains rules and Assignment rules are forbidden in resource definitions. Resources are also prohibited from constraining elements inherited from a parent definition.
+
+**Example:**
+
+* Define a resource representing an emergency vehicle:
+
+  ```
+  Logical:        EmergencyVehicle
```

Comment:
Should this be `Resource: `?

Reply:
Mmmm.  Maybe... 

Look!  A squirrel!  Over there!

Reply:
If I (hypothetically) were to use the `Logical` keyword in the only example I have of a `Resource`, I would (hypothetically) fix it in some commit like 8996a60.

Reply:
@cmoesel and @markkramerus it looks like the `Logical: ` keyword made its way back. Is that intentional? It doesn't look like there are any other examples for `Resource: ` in the documentation.

Reply:
It looks like it somehow snuck back in in a merge commit.  I'll submit a PR to fix it back.

Reply:
I think Mark and I were both making edits at the same time and some wires got crossed.  I'll submit a new PR to fix it.